examples/prism-markup.html

```diff
@@ -72,6 +72,6 @@ <h2>Multi-line attribute values</h2>
 baz"></code></pre>
 
 <h2>XML tags with non-ASCII characters</h2>
-<pre><code>&lt;L&auml;ufer&gt;foo&lt;/L&auml;ufer&gt;
-&lt;tag l&auml;ufer="l&auml;ufer"&gt;bar&lt;/tag&gt;
-&lt;l&auml;ufer:tag&gt;baz&lt;/l&auml;ufer:tag&gt;</code></pre>
\ No newline at end of file
+<pre><code>&lt;Läufer&gt;foo&lt;/Läufer&gt;
+&lt;tag läufer="läufer"&gt;bar&lt;/tag&gt;
+&lt;läufer:tag&gt;baz&lt;/läufer:tag&gt;</code></pre>
```

Comment:
UTF-8 should be supported in this day and age, so there's no need for the HTML entities.